lib/ZumoOta/src/BootloaderCom.cpp

```diff
@@ -433,14 +712,12 @@ void BootloaderCom :: enterBootloader()
 void BootloaderCom::exitBootloader()
 {
     LOG_INFO(" bootloader mode is exited");
-
 }
 
 bool BootloaderCom::process()
 {
     size_t newBytes = 0; 
-    uint8_t receiveBuffer[8];
-
+    uint8_t receiveBuffer[chunkSize];

```

Comment:
mybuffer, m_firmwarebuffer, receive buffer .....

You need to explain why you need all fo them

Reply:
Okay!